src/iarc7_safety/SafetyClient.cpp

```diff
@@ -0,0 +1,92 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// SafetyClient
+//
+// Class implements an easy way to use iarc7_safety to notify of safety events
+//
+////////////////////////////////////////////////////////////////////////////
+
+// Associated header
+#include "iarc7_safety/SafetyClient.hpp"
+
+// ROS message headers
+
+using namespace Iarc7Safety;
+
+SafetyClient::SafetyClient(ros::NodeHandle& nh, const std::string bond_id) :
+bond_id_(bond_id),
+bond_("bond_topic", bond_id_, boost::bind(&SafetyClient::onBroken, this), boost::bind(&SafetyClient::onFormed, this))
```

Comment:
Bind is a member of the standard library now so I'll include #include <functional> and change the namespace to std. 

The reason it works now is that ros uses boost extensively and probably #includes it at some point.

We shouldn't need to declare compile/run dependencies on boost since it's part of the standard library.